PostgreSQL/postgresql.py

```diff
@@ -190,6 +190,23 @@ def table_exists(self, table):
                               "WHERE table_name=%s;", [table])
         return self.fetchone()[0] != 0
 
+    def column_exists(self, table, column):
+        """
+        Test whether the specified SQL column exists in the specified table.
+        :param table: table name to check.
+        :type table: str
+        :param column: column name to check.
+        :type column: str
+        :returns: True if the column exists, False otherwise.
+        :rtype: bool
+        """
+        self.__cursor.execute(
+            "SELECT COUNT(*) FROM information_schema.columns "
+            "WHERE table_name = %s AND column_name = %s", 
```

Comment:
Does postgresql have this convention:

```suggestion
            "WHERE table_name = ? AND column_name = ?", 
```

Reply:
Yes, that's what psycopg2 uses - it's like that in the rest of the addon as well.